notebooks/image_models/labs/autoencoder_anomaly_detection.ipynb

```diff
@@ -0,0 +1,750 @@
+{
```

Comment:
<ul><li>Add a markdown cell above with a "<strong>Exercise</strong>: &lt;description of the task asked&gt;**"  to match with other lab exercise style</li><li>Just typing adding the argument seems a little too easy. What about asking to fill up all the layers? e.g.: <code>encoder = tf.keras.Sequantial([ #TODO])</code></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/215/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>